### ISSUE TITLE
AttributeError: 'AlexaNotificationService' object has no attribute 'email'

### ISSUE THREAD

**Issue by poudenes** [custom-components/alexa_media_player#683]
Yesterday evening it worked but now its broken again.
What I did yesterday:

1) Removed integration
2) Removed Alexa Media Player in HACS
3) Restart HA
4) Add Alexa Media Player in HACS
5) Add Integration
6) Give my Email, Password, did not give a include or exclude. 
7) 3 Devices where add: "This Device" "Peter's Alexa Apps" and "Peter's Echo Dot"
8) All entities are correct.

Today whole day no notifications at all, but when sunset start first notify must start. And didn't hear anything and saw this error in log.

```
Created a automation (when turn on a switch it must notify something)
###########################################################################################
#
# PLAY TEST AND SEE IF THINGS WORK OR NOT
#
###########################################################################################

- alias: "Test - Notify Alexa On"
  trigger:
    platform: state
    entity_id: switch.newkaku_014e5906_a
    to: "on"

  action:
    - service: notify.alexa_media
      data:
        data:
          type: announce
        message: "test"
        target: media_player.peter_s_echo_dot
```

**System details**
 - Home-assistant (version): 0.109.2
- Alexa Media Player Add-On: Installed: v2.6.1

**Logs**
```
2020-05-04 22:01:41 ERROR (MainThread) [homeassistant.components.automation] Test - Notify Alexa On: Error executing script. Unexpected error for call_service at pos 1: 'AlexaNotificationService' object has no attribute 'email'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 153, in _async_step
    self, f"_async_{cv.determine_script_action(self._action)}_step"
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 623, in _async_call_service_step
    *self._prep_call_service_step(), blocking=True, context=self._context
  File "/usr/src/homeassistant/homeassistant/core.py", line 1247, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1282, in _execute_service
    await handler.func(service_call)
  File "/usr/src/homeassistant/homeassistant/components/notify/__init__.py", line 119, in async_notify_message
    await notify_service.async_send_message(**kwargs)
  File "/config/custom_components/alexa_media/notify.py", line 209, in async_send_message
    self.email
AttributeError: 'AlexaNotificationService' object has no attribute 'email'
```

**Additional context**
Yesterday did some tests, notify worked, but now get this error again. 
No change on the Add-On...

**Comment by alandtse** [comment]
That was fixed in 2.6.1. Make sure you're actually on it. The line it's reporting isn't in the [code](https://github.com/custom-components/alexa_media_player/blob/dev/custom_components/alexa_media/notify.py#L209) for 2.6.1.

Duplicate of #666

**Comment by poudenes** [comment]
Thanks, removed integration again, removed HACS again. restart HA, add HACS, add integration and did the test. Alexa now say the message. I'll keep a eye on it.
